src/main/java/com/spartronics4915/frc2020/commands/DriveCommands.java

```diff
@@ -32,8 +34,45 @@ public TeleOpCommand(Joystick joy)
         @Override
         public void execute()
         {
-            mDrive.arcadeDrive(mJoystick.getY(), mJoystick.getX()); // To invert joystick controller,
-                                                                // so forward... is forward
+            // To invert joystick controller, so forward... is forward
+            mDrive.arcadeDrive(mJoystick.getY(), mJoystick.getX());
+        }
+    }
+
+    public class TeleOpInvertedCommand extends CommandBase
+    {
+        private final Joystick mJoystick;
+
+        public TeleOpInvertedCommand(Joystick joy)
+        {
+            mJoystick = joy;
+            addRequirements(mDrive);
+        }
+
+        @Override
+        public void execute()
+        {
+            // Inverted X and Y
+            mDrive.arcadeDrive(-mJoystick.getY(), -mJoystick.getX());
```

Comment:
i'm not sure if this works... If the normal forward driving, it's my recollection that we need to negate y since push-forward is negative (i think).  If that's true, then shouldn't the y be positive in the reversed case?

Reply:
also: we used to have to specify a dead-band.  Is this no-longer needed?

Reply:
A nice feature of the Attack 3 joysticks that Chris uses is a built-in deadband. Will didn't use them in 2018 and we had to specify one.